### ISSUE TITLE
Use NumberControl?

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/fourier-making-waves#80]
From https://github.com/phetsims/fourier-making-waves/issues/72#issuecomment-856223419, related to the Continuous screen:

> - [ ] All 4 sliders have labels that change based on whether "Function of" is "space" or "time".  That makes using NumberControl relatively complicated.  So far, I've made the decision NOT to use NumberControl here.  There's a small trade-off for PhET-iO and (perhaps) a11y.

**Comment by pixelzoom** [comment]
I spent a couple of hours investigating this, bailed and didn't commit anything. Conclusions:

These controls need to change their symbol and units based on domain. NumberControl provides no API for changing the symbol or units. NumberDisplay has `recomputeText`, but that's a bit of hackery, and not accessible via NumberControl.

If NumberDisplay did have a way of observing symbol and units, it can't change them all at once. There would be intermediate states.

Options:
(1) Leave as is, with a custom control for Fourier.
(2) Minor (?) changes to NumberControl, with some hackery in a subclass to make this work.
(3) Major changes to NumberControl - which I do not have time for.

**Comment by pixelzoom** [comment]
I went with option (2) -minor changes to NumberControl, with some hackery in a subclass to make this work.  First, I added NumberControl.redrawNumberControl, to update the display when  the symbol and units change (to match the domain).   Then I converted my classes to NumberControl.  They take up a bit more vertical space (see screenshot below) but seems to be working OK.  And keyboard traversal now shows a focus highlight on the entire NumberControl, instead of just the slider thumb.

Closing.

<img width="262" alt="screenshot_1054" src="https://user-images.githubusercontent.com/3046552/125352396-6f413300-e31e-11eb-8023-7552fd2604f1.png">
